src/ai/backend/manager/models/domain.py

```diff
@@ -17,6 +20,8 @@
 from .scaling_group import ScalingGroup
 from .user import UserRole
 
+log = BraceStyleAdapter(logging.getLogger('ai.backend.gateway.admin'))
```

Comment:
Usually it's sufficient to use `__file__`. Is there specific reasons to manually set this as the path for a different module?

Reply:
Replaced it with `__file__`.